test/stepDefinitions/definitionFileSteps.js

```diff
@@ -16,6 +16,38 @@ defineSupportCode(function ({ Given, When, Then, Before, After }) {
     Data.optionalFields = [{fieldType: 'text', fieldId: 'TextField'}];
   }
 
+  async function populateCaseFieldsWithFileUpload(extension){
+    Data.jurisdiction = 'Auto Test 1';
+    Data.caseType = 'All Field Data Types';
+    Data.event = 'Create a case';
+    Data.optionalFields = [{fieldType: 'text', fieldId: 'TextField'},
+      {fieldType: 'document', value: '\\test\\resources\\dm-store\\dm-store' + extension, fieldId: 'DocumentField'}];
```

Comment:
not sure `\\` will work on the pipeline. please see if using `/` will work for your windows local, if so use that. if it doesn't then try using some sort of file separator function. A quick look online and i found this for node which may work

`path.sep ` note would need to `require('path')
` at the top of the file

Reply:
Done.